### DISCUSSION TITLE
Welcome to kivyauth Discussions!

### DISCUSSION THREAD

**Discussion by shashi278** [shashi278/social-auth-kivy#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jamanz** [comment]
Hello, I wonder if it possible to modify your google sign up code to add additional scopes, for Google Drive as example. Currently I writing kivy android app which have to upload user data to  users Google Sheet after, he sign in and allow access to google drive.
